Automattic/jetpack
# Re-order Sync default option whitelist (allowlist)

The option whitelist (allowlist) used in Jetpack Sync has been appended to over the years leaving the settings no longer grouped meaningfully. This PR aligns them based on alphanumeric order with the exception I didn't distinguish between - or _.

#### Changes proposed in this Pull Request:
* No functional change, planning on adding addtional options in another PR so doing a little cleanup first.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
N/A

#### Testing instructions:
Regression testing that options still sync. Note that we have unit tests that cover default options as well.
* Modify an option from the $default_options_whitelist list
* Verify it is updated in the cache site 

Additionally you will see that the changes are 1-1 additions/deletions.

#### Proposed changelog entry for your changes:
* No Changelog entry needed.
